src/app/components/modules/Powerdown.jsx

```diff
@@ -112,7 +112,7 @@ class Powerdown extends React.Component {
             );
         }
         if (notes.length === 0) {
-            let AMOUNT = vestsToSpf(this.props.state, new_withdraw) / 13;
```

Comment:
I think this place we should replace `let` to `const`.

Reply:
Just replacing `let` with `const` here would not work, because `AMOUNT` is being changed in the next line.
We then would need two constants (AMOUNT, AMOUNT_TEXT).

~~~
const AMOUNT = vestsToSpf(this.props.state, new_withdraw) / 4;
const AMOUNT_TEXT = AMOUNT.toFixed(AMOUNT >= 10 ? 0 : 1);
~~~
This would be an option.